_posts/2024-06-11-quarkus-in-a-foundation.adoc

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: 'Moving Quarkus to a Foundation'
+date: 2024-06-11
+tags: announcement
+synopsis: 'We intend to move Quarkus to a foundation to accelerate the adoption rate, enhance transparency, foster collaboration, and encourage multi-vendor participation and execution'
+author: maxandersen
+---
+
+== tl;dr
+We intend to move Quarkus to a foundation to accelerate the adoption rate, enhance transparency, foster collaboration, and encourage multi-vendor participation and execution. 
+
+== Introduction
+
+The Quarkus project has soared to incredible heights and maturity since its inception. Quarkus was unveiled as a community project in March 2019, followed by the release of Red Hat Build of Quarkus in April 2020. Quarkus has captivated the developer community with over 700 publicly available extensions and close to 1 million downloads of its core artefacts monthly. Quarkus has proven to be a game-changer in the world of Java-based development.
+
+== Community Growth
+
+The vibrant and ever-expanding Quarkus community is a testament to the project’s success. With over 900 contributors, the energy and passion are palpable. In the past year alone, 58% of pull requests to Quarkus core came from 111 non-Red Hat contributors, and 83% of pull requests to the Quarkiverse were from 394 non-Red Hat contributors. This broad and diverse participation emphasises the immense interest and support for Quarkus, showcasing the power of collaborative innovation.
+
+== Vision for Quarkus
+As Quarkus adoption increases, we aim to establish Quarkus as the de-facto standard for new Java-based applications. We are excited to welcome more developers and vendors from across the software ecosystem, inviting them to be part of this dynamic journey.
+
+== Enhancing Governance and Execution
+We are committed to improving our governance and execution by enabling more transparency and participation. Over the coming weeks and months, we will roll out initiatives designed to:
+
+Increase Transparency:: We will provide more visibility into our decision-making processes, development roadmaps, and project status updates to the community.
+Enhance Participation:: We will create more opportunities for community members to get involved in project governance, such as contributing to working efforts, joining meetings/chats, and participating in design discussions.
```

Comment:
What do you mean by `working efforts`?

Reply:
It could be anything around the project that the team currently does.

Reply:
OK, not familiar with the expression, thanks!